daemon/cluster/cluster.go

```diff
@@ -68,7 +68,7 @@ const (
 	swarmConnectTimeout            = 20 * time.Second
 	swarmRequestTimeout            = 20 * time.Second
 	stateFile                      = "docker-state.json"
-	defaultAddr                    = "0.0.0.0:2377"
+	defaultAddr                    = "tcp://0.0.0.0:2377"
```

Comment:
Previously, the default address wasn't validated, and validation requires a full / valid address, so updated the default.